src/components/views/rooms/MemberList.js

```diff
@@ -293,9 +306,12 @@ module.exports = React.createClass({
                     if (memberEvent) {
                         return;
                     }
+                    var avatarJsx = (
+                        <BaseAvatar name={e.getContent().display_name} width={36} height={36} />
```

Comment:
Why are we specifying a custom avatarJsx here? It looks like EntityTile's default avatar generator does the right thing?